import_export_extensions/templates/admin/import_export_extensions/progress_bar.html

```diff
@@ -0,0 +1,3 @@
+{% load i18n %}
+
+<progress max="100" id="progress-bar" data-url="{{ job_url }}" data-label={% trans "Waiting..." %}></progress>
```

Comment:
```suggestion
<progress max="100" id="progress-bar" data-url="{{ job_url }}" data-label={% translate "Waiting..." %}></progress>
```

So I guess this package will help https://github.com/adamchainz/djade in such cases

Reply:
I think we can leave `trans` as it is everywhere else in the project, and then add `djade` later